style.css

```diff
@@ -340,125 +269,103 @@ button:disabled {
 }
 
 .s-button {
-  /* Auto layout */
   align-self: flex-start;
-  justify-self: flex-start;
-  margin-left: 24px;
-  margin-right: auto;
-  width: hug (124px);
-  height: hug (48px);
   padding: 12px;
-  gap: 10px;
   background: #ff6b00;
-  border: none; /* Remove the border */
-  outline: none; /* Remove the outline */
+  border: none;
+  outline: none;
   font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: 700;
   font-size: 17px;
   line-height: 24px;
   text-align: center;
   color: #fff;
-
-  /* Inside auto layout */
-
-  flex: none;
-  order: 3;
-  flex-grow: 0;
+  order: 1;
 }
 
-.art-data {
+.samples {
+  position: relative;
   display: grid;
   grid-template-columns: 1fr;
   grid-gap: 24px;
   align-items: center;
   justify-items: center;
-  position: relative;
-  background-image: url("./images/recent_works/bg-img.png");
-
-  /* background-size: cover; */
+  margin: 24px 24px 120px 24px;
+}
 
-  /* background-position: center; */
+.art-data {
+  position: relative;
+  background-image: url("./assets/background/bg_img.png");
+  background-size: cover;
+  background-position: center;
 }
 
 .overlay-content {
   position: absolute;
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  justify-items: center;
+  width: 100%;
+  height: 100%;
+}
```

Comment:
- **[OPTIONAL]** The hover property in the card lets us see the `see project` button and the picture of the project that is different for each one otherwise the card has a gray coverture, Please 🙏🏽 correct this behavior to be similar to the Figma template 👇🏽 

 | Figma template | Your design |
|---|---|
| ![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/58642949/f450d2dd-5b2b-4bfe-8a32-2168e58601c4) | ![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/58642949/4dc4b584-632b-4263-8277-59c276e8d538) |